docs/docs/10.8-pure-render-mixin.md

```diff
@@ -21,24 +21,18 @@ React.createClass({
 });
 ```
 
-Example using ES6 class syntax:
+Under the hood, the mixin implements [shouldComponentUpdate](/react/docs/component-specs.html#updating-shouldcomponentupdate), in which it compares the current props and state with the next ones and returns `false` if the equalities pass.
 
-```js
-import PureRenderMixin from 'react-addons-pure-render-mixin';
-class FooComponent extends React.Component {
-  constructor(props) {
-    super(props);
-    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind(this);
-  }
+When using ES6 class syntax, you can instead use the `React.PureComponent` base class, which achieves the same result:
```

Comment:
I think we'd also need to mention that `PureComponent` isn't exactly the same as pure-render-mixin, since it also affects the behavior of functional children. See https://github.com/facebook/react/pull/6914